lib/react.dart

```diff
@@ -359,6 +362,35 @@ abstract class Component {
   dynamic render();
 }
 
+/// Creates ReactJS [ReactElement] instances.
```

Comment:
`ReactElement` isn't valid here, since those are only used for non-server runtimes.